packages/translation-extension/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+import { ITranslator } from '@jupyterlab/translation';
+import { INotebookShell } from '@jupyter-notebook/application';
+
+const translator: JupyterFrontEndPlugin<void> = {
```

Comment:
Wondering if we need to create a new package for this plugin.

Or maybe we put the plugin in `application-extension` directly to keep things simpler: https://github.com/jupyter/notebook/blob/main/packages/application-extension/src/index.ts